app.js

```diff
@@ -0,0 +1,44 @@
+const { request } = require("http");
+
+class App {
+  commands = [];
+
+  registerCommand(name, callback) {
+    const object = { name: name, callback: callback };
```

Comment:
https://alligator.io/js/object-property-shorthand-es6/
```suggestion
    const object = { name, callback };
```